web/app/js/components/util/ApiHelpers.jsx

```diff
@@ -147,7 +147,11 @@ const ApiHelpers = (pathPrefix, defaultMetricsWindow = '1m') => {
   };
 
   const fetchExtension = name => {
-    return apiFetch(`${l5dExtensionsPath}?extension_name=${name}`);
+    let extensionPath = l5dExtensionsPath;
+    if (name) {
+      extensionPath += `?extension_name=${name}`;
+    }
+    return apiFetch(extensionPath);
```

Comment:
I think the name might be a bit misleading here. It seems that when we don't add a query param to the `GET /api/extension` path we return all of the extensions. I'm wondering if this shouldn't be its own handler and its own call.

Perhaps we might also be able to rename the path so the get call goes to `/api/extensions/` in which case `/api/extensions?extension_name=viz` might also make sense. Let me know if this makes sense and what your thoughts around this are.

Reply:
Yea thought about this too but from the comment from the issue it was stated that the api/extension route should be modified to accommodate getting all and single extensions. You can check more info on that on the original issue #5899

Reply:
I agree with @mateiidavid on renaming the endpoint. I think now that we are adding the capability to return a list of extensions, it makes more sense to rename the endpoint to `/api/extensions` as well as the method name. On the other hand, I think it would be ok to use one single handler for both cases since the info we dealing with are similar.